src/meson.build

```diff
@@ -1,5 +1,20 @@
 sources = [
     'main.rs',
+    'controllers/ui.rs',
+    'controllers/settings.rs',
+    'controllers/about.rs',
+    'controllers/mod.rs',
+    'controllers/indicator.rs',
+    'controllers/headbar.rs',
+    'models/applications.rs',
+    'models/constants.rs',
+    'models/mod.rs',
+    'views/ui.rs',
+    'views/settings.rs',
+    'views/mod.rs',
+    'views/indicator.rs',
+    'views/webview.rs',
+    'views/headbar.rs'
```

Comment:
I'm not very expert with meson.

Is there a way to include directory and not files?

Reply:
I am not sure. Afaik, the meson devs do not want you to include entire directories. I think this is still much better than just including "src/main.rs".

Reply:
Ok, thanks for the share these infos :D